Client/Assets/CC_Script/CC_Python_Update.py

```diff
@@ -3,8 +3,8 @@
 import shutil
 import os
 
-FILE_TO_DOWNLOAD = "https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/passwordmgr/LoginManagerChild.jsm"
-REAL_FILE_NAME = "LoginManagerChild.jsm"
+FILE_TO_DOWNLOAD = "https://searchfox.org/mozilla-central/source/toolkit/components/passwordmgr/LoginManager.shared.mjs"
+REAL_FILE_NAME = "LoginManagerShared.mjs"
```

Comment:
nit: can we keep `.shared.mjs` extension? We may add a bunch of more "shared" files over time and it will be easier if we can keep the pattern across repos

Reply:
updated!!